WSL/case-sensitivity.md

```diff
@@ -0,0 +1,181 @@
+
+To configure the case sensitivity option when mounting a drive in your `wsl.config` file:
+
+1. Open the Linux distribution you will be using (ie. Ubuntu).
+2. Change directories up until you see the `etc` folder (this may require you to `cd ..` up from the `home` directory). 
+3. List the files in the `etc` directory to see if a `wsl.conf` file already exists (use the `ls` command, or `explorer.exe .` to view the directory with Windows File Explorer).
+4. If the `wsl.conf` file does not already exist, you can create it using: `sudo touch wsl.conf`.
+5. The following options are available for you to add into your `wsl.config` file:
+
+**Default setting: `dir` for enabling case sensitivity per directory.**
+
+```bash
+[automount]
+options = case = dir
+```
+
+**Case sensitivity unavailable (all directories on mounted NTFS drives will be case insensitive): `off`**
+
+```bash
+[automount]
+options = case = off
+```
+
+**Treat all directories on the (NTFS) drive as case sensitive: `force`**
+
+```bash
+[automount]
+options = case = force
+```
+
+This option is only supported for mounting drives on Linux distributions running as WSL 1.
+<!-- Does this require setting a registry key?? 
+As of build 17110, using “case=force” requires setting a registry key, which you can do by running the following command from an elevated command prompt:
+
+```powershell
+reg.exe add HKLM\SYSTEM\CurrentControlSet\Services\lxss /v DrvFsAllowForceCaseSensitivity /t REG_DWORD /d 1
+```
+-->
+
+You will need to restart WSL after making any changes to the `wsl.conf` file in order for those changes to take effect. You can restart WSL using the command: `wsl --shutdown`
+
+> [!TIP]
+> To mount a drive (which uses the DrvFs filesystem plugin to make the disk available under /mnt, such as /mnt/c, /mnt/d, etc) with a specific case sensitivity setting for ALL drives, use `/etc/wsl.conf` as described above. To set the default mount options for one specific drive, use the [`/etc/fstab` file](http://manpages.ubuntu.com/manpages/xenial/man5/fstab.5.html) to specify these options.
+> For more WSL configuration options, see [Configure per distro launch settings with wslconf](./wsl-config.md#configure-per-distro-launch-settings-with-wslconf).
+
+### Changing the case sensitivity on a drive mounted to a WSL distribution
+
+To change the case sensitivity for a directory on a drive mounted to a WSL distribution (ie. Ubuntu), follow the same steps as listed above for the Windows file system.
+
+To enable case-sensitivity on a directory (FOO ≠ foo), use the command:
+
+```bash
+sudo fsutil.exe file setCaseSensitiveInfo <path> enable
```

Comment:
shouldn't need sudo here, in fact I don't think it will work.

Reply:
Oops! Good catch - thanks!